demo-terraform/aws/wordpress_ec2_alb_rds/iam.tf

```diff
@@ -1,64 +1,65 @@
 
 ## Orchestrator acccess will have an especific user for this lab
 locals {
-    user_name = "user-${random_string.lab_id.result}"
-    policy_name = "pol-${random_string.lab_id.result}"
+  user_name   = "user-${random_string.lab_id.result}"
+  policy_name = "pol-${random_string.lab_id.result}"
 
 }
 
 
 ## Create the user
 resource "aws_iam_user" "orch_user" {
-    name = local.user_name
+  name = local.user_name
 
-    tags = {
-        Name = local.user_name
-        "Lab-ID" = random_string.lab_id.result
-        Description = "Guardicore Orchestrator User"
-    }
+  tags = {
+    Name        = local.user_name
+    "Lab-ID"    = random_string.lab_id.result
+    Description = "Guardicore Orchestrator User"
+  }
 }
 
 ## Create the access keys and print in the console
 resource "aws_iam_access_key" "orch_user" {
```

Comment:
**Accurics remediation**
 
**[MEDIUM]: Ensure there is no IAM user with permanent programmatic access**
 
**Vulnerability**
IAM user with permanent programmatic access/root access keys may lead to data leak.
 
**Remediation**
As a best practice, use temporary security credentials (IAM roles) instead of access keys, and disable any AWS account root user access keys.

source: accurics-